services/web-server/procs.yml

```diff
@@ -12,6 +12,16 @@ scanner:
   schedule: '0 0 * * *'
   deadline: 86400
   command: node src/main.js scanner
+expire-authorization-codes:
+  type: cron
+  schedule: '0 0 * * *'
+  deadline: 86400
+  command: node src/main.js expire-authorization-codes
+expire-access-tokens:
+  type: cron
+  schedule: '0 0 * * *'
```

Comment:
Why just once a day? Why not much more frequently? Doesn't this mean that access tokens and authorization codes (for the above one) could stay around for nearly 24 hours after they expire?

Reply:
I think this is just used for general memory cleanup. Each entry has an `expires` entry that we look at. https://github.com/taskcluster/taskcluster/pull/1202/files#diff-9e4009a92aeb7e9312ee0f512c302100R157-R159

Reply:
@helfi92 Ah ok cool, yeah that looks good. It might be worth renaming these jobs to `clean-up-expired-[access-tokens/auth-code]` or similar, but that's you call.

Reply:
This is pretty typical of how we do stuff: check the expired timestamp on every request, and then periodically clean up expired stuff.  The updated name is probably clearer :)